core/src/trezor/messages.py

```
@@ -3638,6 +3638,20 @@ def __init__(
+    class WriteSEPrivateKey(protobuf.MessageType):
+        private_key: "bytes"
+
+        def __init__(
+            self,
+            *,
+            private_key: "bytes",
+        ) -> None:
+            pass
+
+        @classmethod
+        def is_type_of(cls, msg: Any) -> TypeGuard["WriteSEPrivateKey"]:
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `msg`

(ANN401)